code/game/objects/items/grenades/flashbang.dm

```diff
@@ -26,12 +26,16 @@
 
 //Flash
 	if(M.flash_act(affect_silicon = 1))
-		M.Paralyze(max(200/max(1,distance), 60))
+		M.Paralyze(max(20/max(1,distance), 5))
+		M.Knockdown(max(200/max(1,distance), 60))
+
 //Bang
 	if(!distance || loc == M || loc == M.loc)	//Stop allahu akbarring rooms with this.
-		M.Paralyze(200)
+		M.Paralyze(20)
+		M.Knockdown(200)
 		M.soundbang_act(1, 200, 10, 15)
 	else
 		if(distance <= 1)

```

Comment:
```suggestion
		if(distance <= 1) // Adds more stun as to not prime n' pull (#45381)
```

Wish this was doc'd in code as to why it double dips without me having to look it up